src/istgtcontrol.c

```diff
@@ -904,6 +904,59 @@ exec_snap(UCTL_Ptr uctl)
 	return (UCTL_CMD_OK);
 }
 
+static int
+exec_resize(UCTL_Ptr uctl)
+{
+	const char *delim = ARGS_DELIM;
+	char *arg, *result;
+	int rc = 0, wait_time, io_wait_time;
+	char *name = uctl->setargv[0];
+	char *size = uctl->setargv[1];
+
+	if (uctl->setargcnt >= 3)
+		io_wait_time = atoi(uctl->setargv[2]);
+	else
+		io_wait_time = 10;
+
+	if (uctl->setargcnt >= 4)
+		wait_time = atoi(uctl->setargv[3]);
+	else
+		wait_time = 30;
+
+	uctl_snprintf(uctl, "%s \"%s\" \"%s\" \"%d\" \"%d\"\n",
```

Comment:
As of now we are not these using `io_wait_time and wait_time` should we remove it?